### PR TITLE
Fix "bonus" inflection.

### LINKED ISSUE #107: "Bonus" inflection issue
Ember 2.7 ([twiddle](https://ember-twiddle.com/f945daccb6d0e97b8879df1092a68c32?openFiles=templates.application.hbs%2C)):
```bash
> i.pluralize('bonus')
"bonus"
> i.singularize('bonuses')
"bonuses"
```

From rails console:
```ruby
[4] pry(main)> ActiveSupport::Inflector.pluralize('bonus')
=> "bonuses"
[5] pry(main)> ActiveSupport::Inflector.singularize('bonuses')
=> "bonus"
```

It wasn't immediately clear where to start - are there docs on how to make tweaks to the default rules?

EDIT:
jk, fixed and PR opened.

### PR DESCRIPTION
**Pull request by blimmer**
It's similar to status, alias, etc.
Fixes #107.

### CODE REVIEW

File: tests/unit/inflector-test.js [hunk 1]
```diff
@@ -552,3 +552,15 @@ test('Ember.Inflector.singularize passes same test cases as ActiveSupport::Infle
...
+  var inflector = new Ember.Inflector(Ember.Inflector.defaultRules);
+
+  assert.equal(inflector.singularize('bonuses'), 'bonus');
+});
+
+test('Ember.Inflector.singularize can pluralize "bonus"', function(assert) {
+  var inflector = new Ember.Inflector(Ember.Inflector.defaultRules);
+
+  assert.equal(inflector.pluralize('bonus'), 'bonuses');
+});
```
Comment (blimmer) [inline tests/unit/inflector-test.js]: I didn't want to muddy the ActiveSupport::Inflector tests above, so I opted to make these their own tests. If you prefer another style for testing, let me know. I TDDed this one.

**Comment by stefanpenner** [comment]
LGTM.

@olivia got any cycles to cut a release?

**Comment by fivetanley** [comment]
@stefanpenner i got it

**Comment by stefanpenner** [comment]
ah, thank you @fivetanley !

**Comment by fivetanley** [comment]
Should be released as 1.9.6. Let me know if you have issues. https://github.com/stefanpenner/ember-inflector/tree/v1.9.6

@olivia @stefanpenner I also added a CHANGELOG.md

**Pull request merged**